### DISCUSSION TITLE
verification after installation?

### DISCUSSION THREAD

**Discussion by spayqu** [wazuh/wazuh#15454] [Q&A] [answered]
Hello,
so I have installed a lower version of wazuh on debian 10 buster, using this link:
https://documentation.wazuh.com/4.2/installation-guide/open-distro/all-in-one-deployment/all-in-one.html
and all seems to work fine. I have also attached 1 endpoint client to my home lab and it's detectable, but I would like to get some monitorings. For example when I try to install auditbeat by running "auditbeat setup" I am getting:
>Exiting:` couldn't connect to any of the configured Elasticsearch hosts. Errors: [error connecting to Elasticsearch at http://localhost:9200: Get "http://localhost:9200": EOF]'

What is wrong here? I have tried a lot of configs within elasticsearch, kibana and ossec, was stating 0.0.0.0, localhost, my own server IP.

When I try to run a cluster with command "/var/ossec/bin/cluster_control -l" then I am getting:

> ERROR: Cluster is not running.

how can I verify if everything is fine. I have read the manuals but still having troubles. Also how to verify the installation if it's fine and working?
Thank you.

**Comment by mdiego92** [comment, marked as answer]
Hello @spayqu,

Please run the following commands to verify the current state of your environment and send us the outputs to further analyze the issue:

1) Check if there are any Wazuh Manager errors in the ossec.log file:

`grep -iE "err|warn|crit" /var/ossec/logs/ossec.log`

2) We need to make sure that the Filebeat service is running and is able to connect to Elasticsearch:

`filebeat test output`

3) Check if there are any Filebeat errors:

`grep -iE "err|warn|crit" /var/log/filebeat/filebeat`

4) Check if there are any Elasticsearch errors (replace cluster_name with the name of your cluster):

`grep -iE "err|warn|crit" /var/log/elasticsearch/<CLUSTER_NAME>.log`

Please let me know if you have any remaining questions!
Thank you very much.
